book_haven_api/tests/requests/libraries/books/test_add_book_to_library.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from django.urls import reverse
+from rest_framework.test import APIClient
+from book_haven_api.models import Library, Book
+
+@pytest.fixture
+def Libraries():
+    library1 =  Library.objects.create(
+        id=1,
+        name='Library1',
+        street='1234 Library1 St.',
+        city='Library1 City',
+        state='CA',
+        zip='12345',
+        lat=123.456,
+        lon=123.456,
+    )
+
+    return library1
+
+
+@pytest.mark.django_db
+def test_add_book_to_library(Libraries):
+    print('add_book_to_library_happy_path')
+    library1 = Libraries
```

Comment:
Convention in Python would be to use `library_1` here as a variable name

Reply:
Good to know!